Automattic/sensei
# Add Install button to Extensions screen

### Changes proposed in this Pull Request

* Add an install button to plugins in the Extensions screen (disabled if the plugin is already installed)
* Attach React component handling the installation flow & feedback
* Add REST API endpoint to install an extension and monitor progress 

### Testing instructions

* Open Sensei LMS > Extensions page
* Clicking Install on a free extension should download & activate it, and update it's status in-place
* Clicking Install on a paid extension should open a WooCommerce.com checkout page


### Screenshot / Video

![Extensions install button](https://user-images.githubusercontent.com/176949/92234012-94207a80-eeb1-11ea-9c26-cb3c9a9c9932.png)
